build.jam

```diff
@@ -0,0 +1,29 @@
+# Copyright René Ferdinand Rivera Morell 2023-2024
+# Distributed under the Boost Software License, Version 1.0.
+# (See accompanying file LICENSE_1_0.txt or copy at
+# http://www.boost.org/LICENSE_1_0.txt)
+
+require-b2 5.2 ;
+
+import feature ;
+
+feature.feature boost.nowide.lfs : auto no : optional propagated ;
+
+constant boost_dependencies :
```

Comment:
Any reason for the variable instead of inlining in build/Jamfile?

Reply:
So that's intended for ("others"=)humans, isn't it? Because otherwise I don't see how [this](https://github.com/boostorg/nowide/pull/183/files#diff-322788b77c72dedcc9b4a12bf0233ceb8451b4a6baa6d851173ecd5320ac14aeR21) is consistent with e.g. https://github.com/boostorg/variant2/blob/a4b167b7238781d75df5d6684c9e9d40b26456dc/build.jam#L17 given that are different files

My hesitation is because now when reading/editing the build/Jamfile I need to switch to this to see the definition of a variable used there. For head-only libs it is defined and used in the same file (as the build-Jamfile doesn't exist)

To be clear: I'm not saying this is bad, I just want to understand the motivation why.